numpy/polynomial/chebyshev.py

```diff
@@ -1647,6 +1647,129 @@ def chebfit(x, y, deg, rcond=None, full=False, w=None):
     return pu._fit(chebvander, x, y, deg, rcond, full, w)
 
 
+def chebfit2d(x, y, z, deg, rcond=None, full=False, w=None, max_degree=None):
+    """
+    2d Least squares fit of Chebyshev series to data.
+
+    Return the coefficients of a Chebyshev series of degree `deg` that is the
+    least squares fit to the data values `y` given at points `x`.
+    The fitted polynomial(s) are in the form
+
+    .. math::  p(x, y) = c_00 + c_10 * T_10(x, y) + ... + c_nm * T_nm(x, y),
```

Comment:
You need to write this as `T_{10}` etc, else it renders as 
![image](https://user-images.githubusercontent.com/425260/67670034-f2fc9580-f96a-11e9-93f3-072f64a30c4c.png)

As an aside - is it the case that `T_{10}(x, y) = T_1(x)T_0(y)`? I think this would be clearer, since we don't defined a two-argument `T` anyway.

Reply:
I have no idea about Chebyshev polynomials, but that sounds reasonbable. So I changed it

Reply:
I also changed it in the other polynomial types